packages/devextreme/testing/testcafe/tests/accessibility/autocomplete.ts

```diff
@@ -0,0 +1,64 @@
+import url from '../../helpers/getPageUrl';
+import { clearTestPage } from '../../helpers/clearPage';
+import { defaultSelector, testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+import { Properties } from '../../../../js/ui/autocomplete.d';
+import Autocomplete from '../../model/autocomplete';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'))
+  .afterEach(async () => clearTestPage());
+
+const items = ['A', 'Aa', 'B'];
```

Comment:
It's better to choose more informative item's name as follows
const items = ['Item_1', 'Item_2', 'Item_3'];